build on solaris broken

Hi guys
build on solaris used to work with svn, but with your clone I get:

$ gmake
gcc  -o ctags ada.o args.o ant.o asm.o asp.o awk.o basic.o beta.o c.o clojure.o css.o cobol.o dosbatch.o eiffel.o entry.o erlang.o falcon.o flags.o flex.o fortran.o get.o go.o html.o htable.o jscript.o json.o keyword.o lisp.o lregex.o lua.o lxcmd.o main.o make.o matlab.o objc.o ocaml.o options.o parse.o pascal.o pcoproc.o perl.o php.o python.o read.o rexx.o routines.o ruby.o rust.o scheme.o sh.o slang.o sml.o sort.o sql.o strlist.o tcl.o tex.o tg.o trashbox.o verilog.o vhdl.o vim.o windres.o yacc.o vstring.o  
Undefined                       first referenced
 symbol                             in file
WEXITSTATUS                         lxcmd.o
WIFEXITED                           lxcmd.o
ld: fatal: symbol referencing errors
collect2: error: ld returned 1 exit status
gmake: *** [ctags] Error 1

any clues what to do?


gmake -v      
GNU Make 3.82
Built for i386-pc-solaris2.12

- I know above is latest dev solaris, but I guess it would be the same on S10 or S11, so it seems some compatibility broken (or autoconf doing wrong things, old ./configure in svn does correct thing on solaris)

--- a/configure.ac
+++ b/configure.ac
@@ -336,7 +336,7 @@ fi
 
 AC_CHECK_HEADERS_ONCE([dirent.h fcntl.h stat.h stdlib.h string.h])
 AC_CHECK_HEADERS_ONCE([time.h types.h unistd.h])
-AC_CHECK_HEADERS_ONCE([sys/dir.h sys/stat.h sys/times.h sys/types.h])
+AC_CHECK_HEADERS_ONCE([sys/dir.h sys/stat.h sys/times.h sys/types.h sys/wait.h])
 
 
 # Checks for header file macros

--- a/lxcmd.c
+++ b/lxcmd.c
@@ -46,6 +46,9 @@
 #include <ctype.h>
 #include <stdlib.h>   /* for WIFEXITED and WEXITSTATUS */
 #include <string.h>
+#ifdef HAVE_SYS_WAIT_H
+# include <sys/wait.h> /* for WIFEXITED and WEXITSTATUS */
+#endif
 
 #include "debug.h"
 #include "main.h"

